### PR TITLE
Css edit

### PR DESCRIPTION
**Pull request by ZeynebKoc**

### CODE REVIEW

File: src/Pages/AboutUsPage.svelte [hunk 1]
```diff
@@ -129,28 +131,28 @@
...
         height: 176px;
-        margin-left: 85px;
-        margin-top: -350px;
         font-style: normal;
         color: #fefcf7;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        margin-left: 80px;
+        margin-top: -350px;
```
Comment (erdemalpay) [inline src/Pages/AboutUsPage.svelte]: You have put and img and another div after that which should be on top of each other in UI. To make them you have used negative margin. Instead of this you can remove the img element and add a background image to div.

File: src/Pages/AboutUsPage.svelte [hunk 2]
```diff
@@ -181,8 +185,11 @@
     .commitment-content {
         width: 540px;
         height: 340px;
-        margin-left: 570px;
-        margin-top: -469px;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: space-around;
+        margin-top: -18px;
```
Comment (erdemalpay) [inline src/Pages/AboutUsPage.svelte]: Why is this necessary?

File: src/Pages/AboutUsPage.svelte [hunk 3]
```diff
@@ -193,42 +200,45 @@
...
+        margin-top: -419px;
+    }
+    .quality-content-left {
         width: 540px;
         height: 210px;
-        margin-left: 465px;
-        margin-top: -423px;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
```
Comment (erdemalpay) [inline src/Pages/AboutUsPage.svelte]: Isn't default value already flex-start?

File: src/Pages/AboutUsPage.svelte [hunk 4]
```diff
@@ -193,42 +200,45 @@
...
-        height: 474px;
-        margin-left: 663px;
-        margin-top: -387px;
-        border-radius: 8px;
-    }
     .quality-content {
+        width: 1110px;
+        height: 210px;
+        display: flex;
+        flex-direction: row;
```
Comment (erdemalpay) [inline src/Pages/AboutUsPage.svelte]: This is default value. No need to state it.

File: src/Pages/AboutUsPage.svelte [hunk 5]
```diff
@@ -327,6 +342,5 @@
         font-size: 16px;
         line-height: 26px;
         color: #333d4b;
-        margin-top: -9px;
     }
```
Comment (erdemalpay) [inline src/Pages/AboutUsPage.svelte:316]: Why do we have both part-1 and part-3? Aren't they same?

File: src/components/Footer.svelte [hunk 6]
```diff
@@ -47,20 +47,35 @@
...
-        margin-top: 318px;
         background: #2c343e;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
     }
     .footer-left {
         width: 622px;
         height: 26px;
```
Comment (erdemalpay) [inline src/components/Footer.svelte]: Are these necessary? Can we remove them?

File: src/components/Header.svelte [hunk 7]
```diff
@@ -3,26 +3,42 @@
...
+    }
     .srcLogo {
-        margin-left: 80.1px;
-        margin-top: 43.56px;
+        width: 236px;
+        height: 26px;
+    }
+    .header {
+        width: 1280px;
+        height: 26px;
```
Comment (erdemalpay) [inline src/components/Header.svelte]: We may remove this too. If needed you can put margin.

File: src/components/Header.svelte [hunk 8]
```diff
@@ -31,17 +47,17 @@
...
-        float: right;
-        margin-top: 50px;
-        margin-right: 81px;
+        justify-content: space-between;
     }
     li {
-        display: flex;
-        flex-direction: column;
-        padding-left: 33px;
+        margin-top: -5px;
```
Comment (erdemalpay) [inline src/components/Header.svelte]: Try to avoid negative margins as much as possible.

**Review by erdemalpay — APPROVED** [review]
Much better now, but there are still things that you need to get used to. In general try to avoid putting constant values for margins, especially negative values. Try to use width height with proportional values like %30 etc.

File: src/components/Header.svelte (hunk 8 above)
Comment (ZeynebKoc) [reply src/components/Header.svelte]: Removed

File: src/components/Header.svelte (hunk 7 above)
Comment (ZeynebKoc) [reply src/components/Header.svelte]: I removed the header class.

File: src/components/Footer.svelte (hunk 6 above)
Comment (ZeynebKoc) [reply src/components/Footer.svelte]: Removed

File: src/Pages/AboutUsPage.svelte (hunk 1 above)
Comment (ZeynebKoc) [reply src/Pages/AboutUsPage.svelte]: I updated the image as background

File: src/Pages/AboutUsPage.svelte (hunk 3 above)
Comment (ZeynebKoc) [reply src/Pages/AboutUsPage.svelte:241]: I split the section-commitment into 2 divs and placed them row. I removed the negative margin.

File: src/Pages/AboutUsPage.svelte (hunk 2 above)
Comment (ZeynebKoc) [reply src/Pages/AboutUsPage.svelte]: I split the section-commitment  into 2 divs and placed them row. I removed the negative margin.

File: src/Pages/AboutUsPage.svelte (hunk 3 above)
Comment (ZeynebKoc) [reply src/Pages/AboutUsPage.svelte]: I removed flex-start

File: src/Pages/AboutUsPage.svelte (hunk 4 above)
Comment (ZeynebKoc) [reply src/Pages/AboutUsPage.svelte]: I removed  flex-direction: row

File: src/Pages/AboutUsPage.svelte (hunk 5 above)
Comment (ZeynebKoc) [reply src/Pages/AboutUsPage.svelte:316]: part-3's photo size was not the same as the others. I fixed it by taking the photos into separate section.
